libs/sdk-ui-dashboard/src/presentation/flexibleLayout/DashboardLayoutWidget.tsx

```diff
@@ -190,6 +194,14 @@ export const DashboardLayoutWidget: IDashboardLayoutWidgetRenderer<
 
     const remainingGridWidth = isCustomWidget(widget) ? 0 : getRemainingWidthInRow(item, screen);
 
+    const parentWidthForScreen = parentItemSize?.[screen]?.gridWidth;
+
+    if (parentWidthForScreen && currentSize.gridWidth > parentWidthForScreen) {
+        console.error(
+            `DashboardLayoutWidget: Widget ID: ${widget.identifier} has size ${currentSize.gridWidth} bigger than parent's size ${parentWidthForScreen}!`,
```

Comment:
`DashboardLayoutWidget: Widget ID: ${widget.identifier} has width set to ${currentSize.gridWidth} which is bigger than the parent container's width ${parentWidthForScreen}!`